config/experiment/dqn/cartpole.yaml

```diff
@@ -0,0 +1,32 @@
+# @package _global_
+defaults:
+  - override /services/actor:
+      - dqn
+  - override /services/environment: cartpole
+
+run:
+  class_name: actors.dqn.DQNTraining
+  seed: 618
+
+  # Archiving
+  archive_model: True
```

Comment:
The `archive_model` and `archive_frequency` parameters are currently not used in the training class.